Tasks/PerformActionOnMachineGroup/PerformActionOnMachineGroup.ps1

```diff
@@ -30,6 +31,12 @@ $machineGroup = Get-MachineGroup -machineGroupName $MachineGroupName -filters $F
 $providerName = $machineGroup.Provider.Name
```

Comment:
Provider for Standard machine would be NULL, and then de-reefing Name would also throw another error.